src/features/form/validations.jsx

```diff
@@ -66,9 +66,9 @@ const generateAbsoluteRangeValidators = concept => {
   } = concept;
   let hiAbsoluteRange, lowAbsoluteRange;
 
-  if (hiAbsolute || lowAbsolute) {
-    hiAbsoluteRange = hiAbsolute ? maxValue(hiAbsolute) : undefined;
-    lowAbsoluteRange = hiAbsolute ? minValue(lowAbsolute) : undefined;
+  if ((typeof hiAbsolute !== 'undefined' && hiAbsolute !== null)  || (typeof lowAbsolute !== 'undefined' && lowAbsolute !== null)) {
```

Comment:
We can get rid of this if/then, can't we, we do all testing we need in 70:71 right?  (The original code had an if/then, but I don't think it was needed there either?)

Reply:
@mogoodrich If the lowAbsolute=0 and highAbsolute is null then it would never go inside this loop? So, i think we need this check in both places.

Reply:
I meant can we get rid of the if/then entirely, and just rely on lines 70-72? Or am I missing something?

Reply:
That makes sense @mogoodrich . Thanks!